### DISCUSSION TITLE
Detect Regulatory Elements

### DISCUSSION THREAD

**Discussion by VerisimilitudeX** [VerisimilitudeX/DNAnalyzer#226] [Ideas]
https://en.wikipedia.org/wiki/Regulation_of_gene_expression

Please post below if anyone has ideas about how we could go about detecting regulatory elements in DNA. We would greatly appreciate your help!

**Comment by Lumary2** [comment]
Hi, could you specify what exactly needs to be implemented? :-)

**Comment by VerisimilitudeX** [comment]
Hi @Lumary2. Please allow me some time to update the information to ensure its accuracy. In the meantime, feel free to read the following Wikipedia article: https://en.wikipedia.org/wiki/Cis-regulatory_element
